sqlmesh/core/model/definition.py

```diff
@@ -1784,6 +1785,84 @@ class AuditResult(PydanticModel):
     blocking: bool = True
 
 
+def create_model_blueprints(
```

Comment:
This is more like `create_models_from_blueprints` rather that creating blueprints